packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -835,7 +861,7 @@
         align-items: center;
         width: 100%;
 
-        @include mobile-or-tablet-screen {
+        @include mobile {
```

Comment:
Why you are removing this newly added breakpoint? 👀

Reply:
it was a simple merge conflict its resolved 👍